### PR TITLE
Add Calendar & Retrieve Account Name/Type

### PR DESCRIPTION
**Pull request by Brett09**
Issues:
- Calendar: #115 
- Account name/type: #179 

Notes:
A colour for the calendar can be chosen when creating a new calendar

### CODE REVIEW

File: device_calendar/example/lib/presentation/pages/calendar_add.dart [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                    items: ColorChoice.values
+                        .map((color) => DropdownMenuItem(
+                              value: color,
+                              child: Text(color.toString().split('.').last),
+                            ))
+                        .toList(),
+                  ),
+                ],
+              ),
+              if (Platform.isAndroid) ...[
```
Comment (nickrandolph) [inline device_calendar/example/lib/presentation/pages/calendar_add.dart]: I don't think the ...[ is required here is it?

File: device_calendar/ios/Classes/SwiftDeviceCalendarPlugin.swift [hunk 2]
```diff
@@ -688,5 +753,31 @@ extension UIColor {
...
+                var hexNumber: UInt64 = 0
+
+                if scanner.scanHexInt64(&hexNumber) {
+                    a = CGFloat((hexNumber & 0xff000000) >> 24) / 255
+                    r = CGFloat((hexNumber & 0x00ff0000) >> 16) / 255
+                    g = CGFloat((hexNumber & 0x0000ff00) >> 8) / 255
+                    b = CGFloat((hexNumber & 0x000000ff)) / 255
+                    
+                    self.init(red: r, green: g, blue: b, alpha: a)
+                    return
```
Comment (nickrandolph) [inline device_calendar/ios/Classes/SwiftDeviceCalendarPlugin.swift]: should this return a value?  a couple of lines below it returns nil, so seems weird that this return doesn't actually return anything

File: device_calendar/example/lib/presentation/pages/calendar_add.dart (hunk 1 above)
Comment (Brett09) [reply device_calendar/example/lib/presentation/pages/calendar_add.dart]: argh I need to get a habit of not using the array block `...[]` for a single widget. I'll remove this in the next PR

File: device_calendar/ios/Classes/SwiftDeviceCalendarPlugin.swift (hunk 2 above)
Comment (Brett09) [reply device_calendar/ios/Classes/SwiftDeviceCalendarPlugin.swift]: Yeah tested that it returns a wrong colour if I remove that `return` and build fails if I remove `return nil` on the few line below

**Pull request merged**